src/calculateRentalCost.js

```diff
@@ -3,8 +3,20 @@
  *
  * @return {number}
  */
-function calculateRentalCost(days) {
-  // write code here
-}
+function calculateRentalCost(days) { 
+  const PRICE_PER_DAY = 40; 
+  const LONG_TERM = 7;
+   const LONG_TERM_DISCOUNT = 50; 
+   const SHORT_TERM = 3; 
+   const SHORT_TERM_DISCOUNT = 20;
 
+    let basePrice = days * PRICE_PER_DAY;
```

Comment:
Since `basePrice` is not reassigned, it should be declared with `const` instead of `let`.